troubleshooting/known_issues/kube_multus_pod_in_ImagePullBackOff.md

```diff
@@ -0,0 +1,34 @@
+# Kube-multus pod is in ImagePullBackOff
+
+## Description
+
+There is a known problem where kube-multus pods may fail to re-start due to an ImagePullBackOff error. The multus:v3.1 image will need to be re-tagged in nexus and the kube-multus pod will need to be restarted.
```

Comment:
Nexus should be capitalized throughout this document to be consistent (See how it's used in RELEASE_NOTES.md)